components/common/header/authNav.js

```diff
@@ -83,8 +83,7 @@ const AuthNav = ({ user, onLogout }) => {
             />
           </MenuButton>
           <MenuList backgroundColor='lightRock'>
-            {/* TODO: commenting these out for now in case we decide to add them back, but I agree with Eryn that they aren't needed */}
-            {/* <MenuItem
+            <MenuItem
```

Comment:
can we add the boxes to all of these like you did for settings and logout?

Reply:
Done